src/app/Scenes/Home/Components/RecommendedAuctionsRail.tsx

```diff
@@ -0,0 +1,124 @@
+import { ActionType, ContextModule, OwnerType } from "@artsy/cohesion"
+import { Flex } from "@artsy/palette-mobile"
+import { RecommendedAuctionsRail_artworkConnection$key } from "__generated__/RecommendedAuctionsRail_artworkConnection.graphql"
+import { LargeArtworkRail } from "app/Components/ArtworkRail/LargeArtworkRail"
+import { SectionTitle } from "app/Components/SectionTitle"
+import { RailScrollProps } from "app/Scenes/Home/Components/types"
+import { useItemsImpressionsTracking } from "app/Scenes/Home/Components/useImpressionsTracking"
+import HomeAnalytics from "app/Scenes/Home/homeAnalytics"
+import { navigate } from "app/system/navigation/navigate"
+import { extractNodes } from "app/utils/extractNodes"
+import {
+  ArtworkActionTrackingProps,
+  extractArtworkActionTrackingProps,
+} from "app/utils/track/ArtworkActions"
+import React, { memo, useImperativeHandle, useRef } from "react"
+import { FlatList, View } from "react-native"
+import { graphql, useFragment } from "react-relay"
+import { useTracking } from "react-tracking"
+
+interface RecommendedAuctionsRailProps extends ArtworkActionTrackingProps {
+  title: string
+  artworkConnection: RecommendedAuctionsRail_artworkConnection$key
+  isRailVisible: boolean
+}
+
+export const RecommendedAuctionsRail: React.FC<RecommendedAuctionsRailProps & RailScrollProps> =
```

Comment:
Maybe renaming it to "auction lots" makes sense because the rail displays lots, not auctions.

```suggestion
export const RecommendedAuctionLotsRail: React.FC<RecommendedAuctionsRailProps & RailScrollProps> =
```